.nvmrc

```
@@ -0,0 +1 @@
+18
```

Could we match the Gutenberg node required version? https://github.com/WordPress/gutenberg/blob/trunk/.nvmrc
```suggestion
20
```